Gemfile

```diff
@@ -1,4 +1,9 @@
 source 'https://rubygems.org'
 
+gem 'grom', git: 'https://github.com/mattrayner/grom.git', branch: 'mattrayner/new-grom'
```

Comment:
After discussion, we decided that until we have plugged into the front end and checked that is working as we intended, then we won't release New Grom.